aspnetcore/blazor/globalization-localization.md

```diff
@@ -40,53 +51,165 @@ The preceding field types:
+
+When the culture is Peruvian Spanish (`es-PE`), the rendered component uses day/month date formatting (`7/6`), 24-hour time, and period separators in numbers with a comma for the decimal value (`1.999,69`):
+
+* **Date**: 7/6/2021 6:49:38
+* **Number**: 1.999,69
+
+## Statically set the culture
+
+::: zone pivot="webassembly"
 
 ::: moniker range=">= aspnetcore-5.0"
 
-By default, Blazor WebAssembly carries minimal globalization resources required to display values, such as dates and currency, in the user's culture. Applications that must support dynamically changing the culture should configure `BlazorWebAssemblyLoadAllGlobalizationData` in the project file:
+Set the `BlazorWebAssemblyLoadAllGlobalizationData` property to `true` in the app's project file (`.csproj`):
 
 ```xml
 <PropertyGroup>
   <BlazorWebAssemblyLoadAllGlobalizationData>true</BlazorWebAssemblyLoadAllGlobalizationData>
 </PropertyGroup>
 ```
 
-Blazor WebAssembly can also be configured to launch using a specific application culture using options passed to `Blazor.start`. For instance, the sample below shows an app configured to launch using the `en-GB` culture:
+The app's culture can be set in JavaScript when Blazor starts with the `applicationCulture` option passed to `Blazor.start`. The following example configures the app to launch using the United States English (`en-US`) culture. Add the following `<script>` element after the Blazor script (`blazor.webassembly.js`) loads:
 
 ```html
-<script src="_framework/blazor.webassembly.js" autostart="false"></script>
 <script>
   Blazor.start({
-    applicationCulture: 'en-GB'
+    applicationCulture: 'en-US'
   });
 </script>
 ```
 
-For more information on Blazor startup, see <xref:blazor/fundamentals/startup>.
+The value for `applicationCulture` must conform to the [BCP-47 language tag format](https://www.rfc-editor.org/info/bcp47). For more information on Blazor startup, see <xref:blazor/fundamentals/startup>.
 
-The value for `applicationCulture` should conform to the [BCP-47 language tag format](https://tools.ietf.org/html/bcp47).
+An alternative to setting the culture with `Blazor.Start` is to set the culture in C# code. Set <xref:System.Globalization.CultureInfo.DefaultThreadCurrentCulture?displayProperty=nameWithType> and <xref:System.Globalization.CultureInfo.DefaultThreadCurrentUICulture?displayProperty=nameWithType> in `Program.Main` (`Program.cs`).
 
-If the app doesn't require localization, you may configure the app to support the invariant culture, which is based on the `en-US` culture:
+Add the <xref:System.Globalization?displayProperty=fullName> namespace to `Program.cs`:
 
-```xml
-<PropertyGroup>
-  <InvariantGlobalization>true</InvariantGlobalization>
-</PropertyGroup>
+```csharp
+using System.Globalization;
+```
+
+Add the culture settings before the line that builds and runs the <xref:Microsoft.AspNetCore.Components.WebAssembly.Hosting.WebAssemblyHostBuilder> (`await builder.Build().RunAsync();`):
```

Comment:
This doesn't shows how to call javascript funtion that is used to set the culture